mimic/rest/maas_api.py

```diff
@@ -253,11 +253,17 @@ def create_multiplot_from_metric(metric, reqargs, allchecks):
     """
     Given a metric, this will produce fake datapoints to graph
     This is for the multiplot API call
+
+    Also, if you name a PING check "squarewave", you will get
```

Comment:
This documentation leaves open a lot of questions, but they're things that would be perhaps more obvious if the function were generally documented better.  For example, what makes a check a "PING check"?  I would have at least said a ``` ``"remote.ping"`` check ``` to make it clear that's what the check's `type` ought to be, and said "label" rather than "name" to give a hint that it's the "label" field and not, say, a "name" field.

That said, the real issue here is that `metric`, `reqargs`, and `allchecks` are not documented.  Can you open a follow-up ticket to add `:param:` and `:return:` documentation to this function?